content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md

```diff
@@ -17,13 +17,36 @@ weight: 1
 
 # Corda: Enterprise Edition release notes
 
+## Corda: Enterprise Edition 4.9.2
+
+Corda Enterprise 4.9.2 is a patch release of Corda Enterprise which includes the addition of a new Gradle 7 plugin, security upgrades, and fixes for minor bugs.
+
+* A new version of the Gradle plug-in that works with [Gradle 7](https://docs.gradle.org/7.0/release-notes.html) has been released as part of this patch. The plugin can be found in [Jenkins](https://ci01.dev.r3.com/blue/organizations/jenkins/Corda-Gradle-Build-Plugins%2Fcorda-gradle-plugins/detail/release%2F5.1.0-RC01/1/pipeline/).
```

Comment:
Customers don't have access to Jenkins. Can this section on the Gradle 7 plugin be replaced with something like the following...

As part of this patch a new Gradle plugin is provided that supports CorDapp development using Gradle 7. This is in addition to the existing Gradle support.
The plugin is uploaded to artifactory. 
Kotlin and Java samples demonstrating Gradle7 usage is at https://github.com/corda/samples-kotlin/tree/chrisr3-gradle7 and https://github.com/corda/samples-java/tree/chrisr3-gradle7
A readme describing the Gradle 7 plugin is available at https://github.com/corda/corda-gradle-plugins/tree/release/5.1/cordapp

Reply:
@chrisr3 can you also review above please. Feel free to change/update.